mne/io/bti/bti.py

```diff
@@ -918,10 +928,13 @@ def _read_bti_header(pdf_fname, config_fname, sort_by_ch_name=True):
         ch['name'] = ch_cfg['name']
         ch['coil_trans'] = (ch_cfg['dev'].get('transform', None)
                             if 'dev' in ch_cfg else None)
-        if info['data_format'] <= 2:
-            ch['cal'] = ch['scale'] * ch['upb'] * (ch['gain'] ** -1)
+        if pdf_fname:
+            if info['data_format'] <= 2:
```

Comment:
Can you add an inline comment what `<= 2` implies in this context? Integer type or something?